Expecta/Expecta.h

```diff
@@ -12,3 +12,4 @@ FOUNDATION_EXPORT const unsigned char ExpectaVersionString[];
 
 // Enable shorthand by default
 #define expect(...) EXP_expect((__VA_ARGS__))
+#define expect_fail(...) EXP_expect_fail((__VA_ARGS__))
```

Comment:
Let's do `failure` and make a patch release, if in a month or so there's no major issues, then call it OK